lib/puppet_x/puppetlabs/imagebuilder.rb

```diff
@@ -163,6 +164,11 @@ def determine_if_using_puppetfile
         end
       end
 
+      def determine_if_using_factfile
+        if exists_and_is_file(:factfile)
+          @context[:use_factfile] = true
+        end
+       end
       def determine_if_using_hiera
```

Comment:
I have added a new line after the function and made sure end is aligned with the function def